tg_bot.py

```diff
@@ -250,16 +259,36 @@ def print_unsubscribe_button(msg):
         reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
 
 
+def print_subscription_status(update, msg, context): 
+    """
+    Prints current subscription status
+    """
+    user_id = str(update.effective_user.id)
+
+    # define subscription limit 
+    subscription_limit = scheduled_jobs[user_id].date() + datetime.timedelta(days=7)
+    # format date 
+    date_object = subscription_limit.strftime("%d-%m-%Y")
+
+    # check if exists a schedule job
+    if user_id in scheduled_jobs:
+        msg.reply_text('Current subscription details:\n\n - Department: %s \n\n - Type: %s \n\n - Until: %s \n' % (
+                context.user_data['buro'], context.user_data['termin_type'], date_object))
+
+    return
+
+
 def stop_checking(update, context):
     user_id = str(update.effective_user.id)
     remove_job(user_id)
     return selecting_buro(update, context)
 
 
-def remove_job(user_id):
+def remove_job(user_id, msg):
```

Comment:
This doesn't work like that =) `remove_job` is called by timer and doesn't have any `msg` there. I'd suggest just to remove it for now.

Reply:
I tried with including `update` as parameter and then `msg = update.message` but this didn't work very well. Thus I am going to comment this part, do you have any idea?